tests/integration/test_transaction_integration.py

```diff
@@ -72,7 +85,7 @@ def connection(ddb_url):
     yield Connection(host=ddb_url)
 
 
-@pytest.fixture(scope='module', autouse=True)
+@pytest.fixture(scope='function', autouse=True)
```

Comment:
Why?

Reply:
So that the `Foo` table will be created before and deleted after each of the test cases that I added. No special reasoning here, it was just an easy way to avoid collisions.

Reply:
Can you instead use a random (or incremental) ID for your tests, e.g. `str(uuid4())`? This way you don't need to delete tables between tests.

Reply:
Sure